front/app/api/comments/useMarkCommentForDeletion.ts

```diff
@@ -54,6 +55,9 @@ const useMarkCommentForDeletion = ({
         queryClient.invalidateQueries({
           queryKey: commentsKeys.list({ commentId: parentCommentId }),
         });
+        queryClient.invalidateQueries({
+          queryKey: moderationsKeys.all(),
+        });
```

Comment:
It's just one big list on the back office, so I think we need to invalidate the full list :thinking: